appveyor.yml

```diff
@@ -1,29 +1,112 @@
 environment:
   NewPascalVersion: v1.0.50
 
-install:
-  - appveyor AddMessage -Category Information "Downloading NewPascal %NewPascalVersion%..."
-  - appveyor DownloadFile https://github.com/newpascal/newpascal/releases/download/np-%NewPascalVersion%/newpascal.zip
-  - appveyor AddMessage "Unpacking NewPascal %NewPascalVersion%..."
-  - 7z x -y "newpascal.zip" -o"C:\" > nul
-  - appveyor AddMessage -Category Information "Configuring NewPascal..."
-  - c:\newpascal\configure.bat
-  - set FPCDIR=c:\newpascal\fpcsrc
-  - set PATH=c:\newpascal\fpc\bin\i386-win32;c:\newpascal\binw32;C:\Program Files (x86)\NSIS;%path%
-
-build_script:
-  - appveyor AddMessage -Category Information "Building UltraStar-Deluxe..."
-  - c:\newpascal\lazarus\lazbuild src\ultrastardx-win.lpi --lazarusdir=c:\newpascal\lazarus
-  - appveyor AddMessage -Category Information "Done."
-
-after_build:
-  - appveyor AddMessage -Category Information "Creating Installer..."
-  - xcopy game\*.dll installer\dependencies\dll /y
-  - makensis "installer\UltraStar Deluxe.nsi"
-  - appveyor AddMessage -Category Information "Done."
-
-artifacts:
-  - path: game
-    name: UltraStar.Deluxe_continuous_portable
-    type: zip
-  - path: installer\dist\UltraStar.Deluxe_*_installer.exe
+image:
+  - Visual Studio 2015
```

Comment:
Why is Visual Studio 2015 mentioned here?

Reply:
Because that is the default that has been used with our Windows build on Appveyor for the past five years.
https://www.appveyor.com/updates/2016/03/08/

Reply:
but why would compiling USDX need Visual Studio??? We need Freepascal/Lazarus and that is about all for the Windows targeting build.

Reply:
It does not need Visual Studio. All Windows images provided by AppVeyor are called Visual Studio 20xx.
On https://www.appveyor.com/docs/build-environment/#choosing-image-for-your-builds it says
> If the build configuration does not specify build worker image then Visual Studio 2015 image is used.

I just wrote out what was implicitly always the case because I had to add two Linux images to the list.